tdrs-backend/tdpservice/users/admin.py

```diff
@@ -23,6 +23,7 @@ class UserAdmin(admin.ModelAdmin):
     exclude = ['password', 'user_permissions', 'is_active']
     readonly_fields = ['last_login', 'date_joined', 'login_gov_uuid', 'hhs_id']
     form = UserForm
+    list_filter = ('access_request',)
```

Comment:
@raftmsohani added this list filter so admins can easily see a list of user's pending access approval

Reply:
The flag will never be removed, so it will have to look at the users with no roles.

Reply:
@jorgegonzalez what does it mean for [Miles' account](https://tdp-backend-raft.app.cloud.gov/admin/users/user/d44233b4-82f4-4e7d-b5bd-f28e3f65732e/change/?_changelist_filters=access_request__exact%3D1) to have a green check while also having roles already assigned? (note: i didn't modify this account as part of this review).